src/codeu/chat/server/Server.java

```diff
@@ -143,7 +142,26 @@ public void onMessage(InputStream in, OutputStream out) throws IOException {
       }
     });
 
-    // Get Conversations By Id - A client wants to get a subset of the converations from
+    // Get Server Request - A client whats to get all the request to the server from the back end.
+    this.commands.put(NetworkCode.SERVER_INFO_REQUEST, new Command(){
+        @Override
+        public void onMessage(InputStream in, OutputStream out) throws IOException{
+            Serializers.INTEGER.write(out, NetworkCode.SERVER_INFO_RESPONSE);
+
+            try{
+              versionServerInfo = new VersionServerInfo();
+            }catch (Exception ex){
+              LOG.error(ex, "There was a problem with parsing the VersionServerInfo.");
+            }
+
+            Uuid.SERIALIZER.write(out, versionServerInfo.version);
```

Comment:
This only writes out the version information, correct? What about the time information? Will you be using the uptimeServerInfo variable to access the time?